src/main/java/com/ldtteam/blockout/BOScreen.java

```diff
@@ -63,14 +62,13 @@ public void render(final int mx, final int my, final float f)
     }
 
     @Override
-    public boolean keyPressed(final int key, final int p_keyPressed_2_, final int p_keyPressed_3_)
+    public boolean keyPressed(final int key, final int scanCode, final int modifiers)
     {
-        if (key == GLFW.GLFW_KEY_BACKSPACE || key == GLFW.GLFW_KEY_ESCAPE || key == GLFW.GLFW_KEY_ENTER || key == GLFW.GLFW_KEY_TAB || key == GLFW.GLFW_KEY_BACKSPACE ||
-            key == GLFW.GLFW_KEY_INSERT || key == GLFW.GLFW_KEY_DELETE || key == GLFW.GLFW_KEY_RIGHT || key == GLFW.GLFW_KEY_LEFT || key == GLFW.GLFW_KEY_DOWN ||
-            key == GLFW.GLFW_KEY_UP || key == GLFW.GLFW_KEY_PAGE_UP || key == GLFW.GLFW_KEY_PAGE_DOWN || key == GLFW.GLFW_KEY_HOME || key == GLFW.GLFW_KEY_END ||
-            // key == GLFW.GLFW_KEY_CAPS_LOCK ||
-            // key == GLFW.GLFW_KEY_SCROLL_LOCK ||
-            // key == GLFW.GLFW_KEY_NUM_LOCK ||
+        if (key == GLFW.GLFW_KEY_BACKSPACE || key == GLFW.GLFW_KEY_ESCAPE || key == GLFW.GLFW_KEY_ENTER || key == GLFW.GLFW_KEY_TAB ||
```

Comment:
I know its not new, but think it might be a good idea to just put all those keys into a hashmap and then just check contains?